aosp_diff/preliminary/packages/apps/Settings/03_0003-Fix-Settings-WifiScanModeActivity-Overlaid-issue.patch

```diff
@@ -0,0 +1,44 @@
+From 3ce3f0a163ead5e59fe132244fd1cfe15171bc16 Mon Sep 17 00:00:00 2001
+From: Weng Su <wengsu@google.com>
+Date: Fri, 29 Jan 2021 17:24:51 +0800
+Subject: [PATCH] Fix Settings WifiScanModeActivity Overlaid issue
+
+- Disallowed non system overlay windows
```

Comment:
These 2 lines removed from comments?

Reply:
Those 2 lines are part of commit message:

" From 37231e5e6d4df4035bbcdca744397bee8f6fbbf9 Mon Sep 17 00:00:00 2001
From: Weng Su <wengsu@google.com>
Date: Fri, 29 Jan 2021 17:24:51 +0800
Subject: [PATCH] [Security Report] Fix Settings WifiScanModeActivity Overlaid
 issue

- Disallowed non system overlay windows

- Screenshot
  https://screenshot.googleplex.com/77fJ9QN6pV4zFYc

Bug: 174047492
Test: manual test"